assets/css/translation-events.css

```diff
@@ -174,7 +174,9 @@ span.event-list-date {
 }
 
 /* Event list: Event buttons. */
-.event-list-item-button {
+a.event-list-item-button,
+a.event-list-item-button:visited,
+a.event-list-item-button:hover {
```

Comment:
FWIW there is now a `:any-link` CSS Pseudo class: https://developer.mozilla.org/en-US/docs/Web/CSS/:any-link

Reply:
Tested and works fine in all browsers, including Safari. Thank you!